modules/claims_api/config/schemas/526.json

```diff
@@ -451,6 +451,7 @@
                   "Public Health Service",
                   "Regular Philippine Scout",
                   "Regular Scout Service",
+                  "Space Force",
```

Comment:
I think Kayla mentioned in the Jira ticket that we also need to add `Space Force` to ::
- `servicePay.militaryRetiredPay.payment.serviceBranch`
- `servicePay.separationPay.payment.serviceBranch`